study.md

```diff
@@ -135,24 +135,23 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a breif description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+Transfer protocall to transfer text between computers
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client is and what is a server? How do they interact with each other?
 
-```md
-<!-- your answer here -->
-```
+A client is a user / a requester of a service. A server is a provider of tbe service.
+
```

Comment:
Yes but be careful with the use of service there, only because it will later have other connotations as well. It could also just be information or a resource on a server, you nailed the requestr/response relationship though.